examples/index.md

```diff
@@ -389,127 +342,61 @@ Content-Type: application/vnd.api+json
 {
   "errors": [
     {
-      "title":  "Missing Data Member",
-      "source": { "pointer": "/data" }
+      "source": { "pointer": "/" },
+      "detail":  "Missing `data` Member at document's top level."
```

Comment:
Curious why you chose to use detail, rather than title, and I could have sworn the example of this I saw was `pointer: /data`, so I guess that was not precise?